### PR TITLE
Fix the Develop Build

### PR DESCRIPTION
**Pull request by jeremyk-91**
For some reason, #7149 merged even though the build didn't pass. This should fix some of the issues in that PR.

Separately we should look at why the merge was allowed.

### CODE REVIEW

File: .palantir/revapi.yml [hunk 1]
```diff
@@ -136,6 +136,14 @@ acceptedBreaks:
...
       justification: "Adding getDdlManager() to TransactionManager will not break\
         \ existing usage"
+  "0.1101.0":
+    com.palantir.atlasdb:atlasdb-cassandra:
+    - code: "java.method.parameterTypeChanged"
+      old: "parameter void com.palantir.atlasdb.keyvalue.cassandra.TracingQueryRunner::<init>(===org.slf4j.Logger===,\
+        \ java.util.function.Supplier<com.palantir.atlasdb.cassandra.CassandraTracingConfig>)"
+      new: "parameter void com.palantir.atlasdb.keyvalue.cassandra.TracingQueryRunner::<init>(===com.palantir.logsafe.logger.SafeLogger===,\
+        \ java.util.function.Supplier<com.palantir.atlasdb.cassandra.CassandraTracingConfig>)"
+      justification: "internal API change on tracingqueryrunner"
```
Comment (barisoyoruk) [inline .palantir/revapi.yml:146]: I checked [sourcegraph](https://sourcegraph.palantir.build/search?q=context:global+TracingQueryRunner&patternType=standard&sm=1&groupBy=repo) and this is only used in atlasdb so we should be safe

File: atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/AbstractCassandraKeyValueServiceIntegrationTest.java [hunk 2]
```diff
@@ -255,8 +255,11 @@ private static String getInternalTestTableName() {
     @Test
     @SuppressWarnings("CompileTimeConstant")
     public void shouldNotErrorForTimestampTableWhenCreatingCassandraKvs() {
-        verify(logger, never())
-                .error(startsWith("Found a table {} that did not have persisted"), ArgumentMatchers.<Arg<?>>any());
+        ArgumentCaptor<Arg<String>> argCaptor = ArgumentCaptor.forClass(Arg.class);
+        // A bit strange, but the purpose is to feed the table references to the argument captor.
+        verify(logger, atLeast(0))
+                .error(startsWith("Found a table {} that did not have persisted"), argCaptor.capture());
+        assertThat(argCaptor.getAllValues()).noneMatch(arg -> arg.getValue().contains("timestamp"));
```
Comment (barisoyoruk) [inline atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/AbstractCassandraKeyValueServiceIntegrationTest.java]: Can we try this instead? I think it is clearer:

```
        verify(logger, atLeast(0))
                .error(startsWith("Found a table {} that did not have persisted"), assertArg((Arg<String> arg) ->
                        assertThat(arg.getValue()).doesNotContain("timestamp")));
```

**Review by barisoyoruk — APPROVED** [review]
LGTM. Just nits

File: atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/AbstractCassandraKeyValueServiceIntegrationTest.java (hunk 2 above)
Comment (jeremyk-91) [reply atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/AbstractCassandraKeyValueServiceIntegrationTest.java]: Thanks - I think it's a good call to use `assertArg(...)`, never heard of that before. That said, I think this as written is a bit different: the [docs](https://javadoc.io/doc/org.mockito/mockito-core/latest/org/mockito/ArgumentMatchers.html) say that "matching is considered successful when the consumer given by parameter does not throw an exception." So I think this verifies that the logger, at least 0 times, is called with a string not containing "timestamp", which would always pass.

I _think_ what we want is `verify(logger, never()).error(startsWith(...), assertArg(arg -> assertThat(arg).contains("timestamp"));` actually. I'll test this

Comment (jeremyk-91) [reply atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/AbstractCassandraKeyValueServiceIntegrationTest.java]: oops, never mind - I'm wrong. Thanks for the suggestion again!

**Pull request merged**